### DISCUSSION TITLE
Conditional Installation of .NETFramework 4.8

### DISCUSSION THREAD

**Discussion by sansa20** [wixtoolset/issues#7254] [Questions]
I have a WiX installer and I have created a bootstrapper project for the same in order to install version 4.8 of .NETFramework as my pre requisites. However, I would like pre-requisites to only get installed if it  is not already there on the system.
Additionally, I have a requirement to install the pre-requisite directly from my computer (I have a pre-requisites folder in my file explorer), and not from the internet. Following is the code I have so far under my bundle.wxs file. Any help would be much appreciated.

```
<Fragment>
        <PackageGroup Id="DotNet">
            <ExePackage SourceFile="..\..\..\External\Prerequisites\.NET Framework\ndp48-web.exe"
                        DetectCondition="ExeDetectedVariable"
                        InstallCommand="/q /ACTION=Install"
                        RepairCommand="/q ACTION=Repair /hideconsole"
                        UninstallCommand="/q ACTION=Uninstall /hideconsole"/>
        </PackageGroup>
    </Fragment> 
```

**Comment by robmen** [comment]
Use a correct `DetectCondition` and it will behave exactly as you desire.

**Reply by sansa20** [reply]
I understand that if the detect condition is evaluated as false, only then the package will be installed on the machine. However, I am unsure what value is to be passed inside- is there any official documentation I could use to understand the same?

**Reply by robmen** [reply]
Yes, there is.
